odoo/tools/js_transpiler.py

```diff
@@ -537,10 +537,10 @@ def relative_path_to_module_path(url, path_rel):
 
 
 ODOO_MODULE_RE = re.compile(r"""
-    \s*                                       # some starting space  
+    \s*                                       # some starting space
     \/(\*|\/).*\s*                            # // or /*
     @odoo-module                              # @odoo-module
-    (\s+alias=(?P<alias>\S+))?                # alias=web.AbstractAction (optional)
+    (\s+alias=(?P<alias>[\w.]+))?     # alias=web.AbstractAction (optional)
```

Comment:
don't forget to fix the comment alignment, and probably cover digits in the testcases as well, just in case? :-)

@robodoo delegate+